app/src/main/java/org/p2p/wallet/send/ui/dialogs/SendTransactionsDetailsBottomSheet.kt

```diff
@@ -0,0 +1,157 @@
+import org.p2p.wallet.utils.copyToClipBoard
+import org.p2p.wallet.utils.withArgs
+
+private const val ARG_SEND_STATE = "ARG_SEND_STATE"
+
+class SendTransactionsDetailsBottomSheet : BaseDoneBottomSheet() {
+
+    companion object {
+        fun show(
+            fm: FragmentManager,
+            title: String,
+            state: SendTotal,
+        ) = SendTransactionsDetailsBottomSheet().withArgs(
+            ARG_TITLE to title,
+            ARG_SEND_STATE to state,
+        ).show(fm, SendTransactionsDetailsBottomSheet::javaClass.name)
+    }
+
+    private lateinit var binding: DialogSendTransactionsDetailsBinding
+
+    private val state: SendTotal by args(ARG_SEND_STATE)
+
+    override fun onCreateInnerView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
+        binding = DialogSendTransactionsDetailsBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        with(binding) {
+            setRecipientAddress()
+            setRecipientGets()
+            setTransactionFee()
+            setAccountCreation()
+            setTotal()
+        }
+
+        setDoneButtonVisibility(isVisible = false)
+    }
+
+    private fun DialogSendTransactionsDetailsBinding.setRecipientAddress() {
+        val address = state.recipientAddress
+        val addressIsEmpty = address.isNullOrEmpty()
+        with(itemAddress) {
+            root.isVisible = !addressIsEmpty
+            imageViewIcon.setImageResource(R.drawable.ic_wallet_home)
+            textViewTitle.text = getString(R.string.send_transactions_details_address)
+            textViewSubtitle.text = address
+            itemAddress.root.setOnLongClickListener {
+                requireContext().copyToClipBoard(address.orEmpty())
+                true
+            }
+        }
+    }
+
+    private fun DialogSendTransactionsDetailsBinding.setRecipientGets() {
+        val color = getColor(R.color.text_mountain)
+        with(itemGets) {
+            imageViewIcon.setImageResource(R.drawable.ic_receive)
+            textViewTitle.text = getString(R.string.send_transactions_details_gets)
+            textViewSubtitle.text = SpanUtils.highlightText(
+                state.fullReceive,
+                state.approxReceive,
+                color
+            )
+        }
+    }
+
+    private fun DialogSendTransactionsDetailsBinding.setTransactionFee() {
+        val colorNight = getColor(R.color.text_mountain)
+        val colorMountain = getColor(R.color.text_night)
+        val colorMint = getColor(R.color.text_mint)
+        with(itemTransactionFee) {
+            imageViewIcon.setImageResource(R.drawable.ic_lightling)
+            textViewTitle.text = getString(R.string.send_transactions_details_transaction_fee)
+            textViewSubtitle.apply {
+                when (val fee = state.fee) {
```

Comment:
if

Reply:
there will be BTC in future so i ignore this IF for now